.cargo-husky/hooks/pre-commit

```diff
@@ -13,9 +13,7 @@ fi
 # Set -x  here to avoid printing secrets
 set -xe
 
-# Setting RUSTFLAGS env for clippy makes it not include custom rules
-RUSTFLAGS=-Dwarnings cargo check --workspace --all-targets ${ALL_FEATURES}
-cargo clippy --all --all-targets ${ALL_FEATURES} -- -D warnings
```

Comment:
I've removed the separate `cargo check` as Clippy is running it anyway and it was triggering recompilation for some reason.